www/source/docs/contribute-help-build.html.md

```diff
@@ -12,7 +12,7 @@ The remainder of this document describes some of the areas of the product that c
 
 ## More Topologies
 
-Habitat ships on day one with [two topologies](/docs/run-packages-topologies): standalone, and leader-follower. There are undoubtedly more that are needed in order to represent the semantics of real systems.
```

Comment:
I certainly dont think this is blocking but technically I don't think this comma is necessary/correct. You would want it if the clause after the and could be a complete sentance or if it was a list of atleast 3 (this is controversial). That all said I'm totally ready to be told I'm wrong and honestly I don't care passionately about this :)